metrics-core/src/main/java/com/yammer/metrics/core/Histogram.java

```diff
@@ -36,14 +36,13 @@ public Sample newSample() {
         },
 
         /**
-         * Uses an exponentially decaying sample of 1028 elements, which offers a 99.9% confidence
-         * level with a 5% margin of error assuming a normal distribution, and an alpha factor of
-         * 0.015, which heavily biases the sample to the past 5 minutes of measurements.
+         * Uses an array of {@link UniformSample} that get cleared periodically so that data can't
+         * hang around for more than a few minutes.
          */
         BIASED {
             @Override
             public Sample newSample() {
-                return new ExponentiallyDecayingTimeWindowedSample();
+                return new UniformTimeWindowedSample(DEFAULT_SAMPLE_SIZE, DEFAULT_SAMPLE_COUNT);
```

Comment:
See https://github.com/HubSpot/metrics/pull/7#issuecomment-254221355 for why this probably isn't what we want.